app/models/gem_typo.rb

```diff
@@ -5,41 +5,37 @@ class GemTypo
 
   include Gem::Text
 
-  DOWNLOADS_THRESHOLD = 10_000_000
-  SIZE_THRESHOLD = 4
-
   def initialize(rubygem_name)
-    @rubygem_name       = rubygem_name.downcase
-    @distance_threshold = distance_threshold
+    @rubygem_name = rubygem_name
   end
 
   def protected_typo?
-    return false if @rubygem_name.size < GemTypo::SIZE_THRESHOLD
-
-    gem_typo_exceptions = GemTypoException.all.pluck(:name)
-    return false if gem_typo_exceptions.include?(@rubygem_name)
```

Comment:
I think we would want to keep the ability to manually whitelist protect gem names.

Reply:
Good catch.